VAMobile/src/screens/HealthScreen/Pharmacy/RefillScreens/RefillScreen.test.tsx

```diff
@@ -36,26 +28,17 @@ context('RefillScreen', () => {
         ...errorState,
       },
     }
-
-    component = render(<RefillScreen {...props} />, { preloadedState: store })
-    testInstance = component.UNSAFE_root
+    render(<RefillScreen {...props} />, { preloadedState: store })
   }
 
-  it('initializes correctly', async () => {
-    await waitFor(() => {
-      initializeTestInstance()
-    })
-    expect(component).toBeTruthy()
-  })
-
   describe('no there are no refillable prescriptions', () => {
     it('should show NoRefills component', async () => {
       await waitFor(() => {
```

Comment:
This `waitFor` can probably be removed, along with the other one in this file

Reply:
updated